src/modules/auth/actions/use-unlocked-account.js

```diff
@@ -1,14 +1,15 @@
 import { augur } from 'services/augurjs'
 import { updateIsLoggedAndLoadAccountData } from 'modules/auth/actions/update-is-logged-and-load-account-data'
 import isMetaMask from 'modules/auth/helpers/is-meta-mask'
+import isGlobalWeb3 from 'modules/auth/helpers/is-global-web3'
 import logError from 'utils/log-error'
 
 const { ACCOUNT_TYPES } = augur.rpc.constants
 
 // Use unlocked local account or MetaMask account
 export const useUnlockedAccount = (unlockedAddress, callback = logError) => (dispatch) => {
   if (unlockedAddress == null) return callback('no account address')
-  if (isMetaMask()) {
+  if (isGlobalWeb3() || isMetaMask()) {
```

Comment:
I think `if (isGlobalWeb3())` should work fine here; `isMetaMask()` actually does `isGlobalWeb3()` inside of it, so the `||` will never be hit!